src/network/ledger.rs

```diff
@@ -838,38 +838,17 @@ impl<N: Network, E: Environment> Ledger<N, E> {
 
         // Case 2 - Proceed to send block requests, as the peer is ahead of this ledger.
         if let Some((peer_ip, is_fork, maximum_block_locators)) = maximal_peer {
-            // Determine the common ancestor block height between this ledger and the peer.
-            let mut maximum_common_ancestor = 0;
-            // Determine the first locator (smallest height) that does not exist in this ledger.
```

Comment:
Can you preserve these comments in the new `verify_block_hashes` method at the same place for readability?